frontend/controller/actions/identity.js

```diff
@@ -350,11 +340,12 @@ export default (sbp('sbp/selectors/register', {
       console.error(`updateLoginState: ${e.name}: '${e.message}'`, e)
     }
   },
-  'gi.actions/identity/login': async function ({ username, password }: {
-    username: string, password: ?string
+  'gi.actions/identity/login': async function ({ username, password, identityContractID }: {
+    username: ?string, password: ?string, identityContractID: ?string
   }) {
-    // TODO: Insert cryptography here
-    const identityContractID = await sbp('namespace/lookup', username)
+    if (username) {
```

Comment:
Under what circumstances would `username` not be set and why?

Please add a comment explaining

Reply:
When the page is refreshed, and because now the values are indexed by username.